tests/scene/test_text_edit.h

```diff
@@ -3139,7 +3139,7 @@ TEST_CASE("[SceneTree][TextEdit] viewport") {
 	v_scroll = text_edit->get_v_scroll();
 	SEND_GUI_MOUSE_BUTTON_EVENT(text_edit, Point2i(10, 10), MouseButton::WHEEL_DOWN, MouseButton::WHEEL_DOWN, Key::NONE);
 	text_edit->notification(TextEdit::NOTIFICATION_INTERNAL_PHYSICS_PROCESS);
-	CHECK(text_edit->get_v_scroll() > v_scroll);
+	CHECK(text_edit->get_v_scroll() >= v_scroll);
```

Comment:
Nitpick, but this commit should likely come before the SceneTree change in the commit history, otherwise it means we have 2 commits with failing tests that would end up merged.